.travis.yml

```diff
@@ -22,6 +22,7 @@ install:
    - arduino --install-library "VL53L0X"
    - arduino --install-library "ESP32 AnalogWrite"
    - arduino --install-library "ServoESP32"
+   - arduino --install-library "SimpleTimer.h"
```

Comment:
If every other dependency is pulled without any mention to a `.h` file, why would this one?

Also, [reading the documentation on how to add libraries not included in the library manager](https://github.com/adafruit/ci-arduino#using-external-libraries) will help you.

Reply:
yes this is why I should not code at night.. solved now